tests/budget_accounting_test.py

```diff
@@ -160,6 +160,33 @@ def test_n_aggregations(self, n_aggregations):
 
         budget_accountant.compute_budgets()
 
+    def test_aggregation_weights(self):
```

Comment:
Do we also test automatic budget splitting, i.e. equal weights when weights are not specified, somewhere?

Reply:
`weights` in BudgetAccountant is not optional parameter, it has to be specified